setup_dependencies.py

```diff
@@ -138,35 +138,35 @@
                     'WMCore.ReqMgr.DataStructs.RequestStatus',
                     'WMCore.ReqMgr.DataStructs.RequestType'
                     ],
-        'systems': ['wmc-rest', 'wmc-database'],
+        'systems': ['wmcrest', 'wmcdatabase'],
         'statics': [],
     },
-    'reqmgr2ms-unmerged': {
+    'msunmerged': {
         'packages': ['WMCore.MicroService.MSUnmerged+'],
-        'systems': ['reqmgr2ms-core'],
+        'systems': ['mscore'],
     },
-    'reqmgr2ms-output': {
+    'msoutput': {
         'packages': ['WMCore.MicroService.MSOutput+'],
-        'systems': ['reqmgr2ms-core'],
+        'systems': ['mscore'],
     },
-    'reqmgr2ms-pileup': {
+    'mspileup': {
         'packages': ['WMCore.MicroService.MSPileup+'],
-        'systems': ['reqmgr2ms-core'],
+        'systems': ['mscore'],
     },
-    'reqmgr2ms-transferor': {
+    'mstransferor': {
         'packages': ['WMCore.MicroService.MSTransferor+'],
-        'systems': ['reqmgr2ms-core'],
+        'systems': ['mscore'],
     },
-    'reqmgr2ms-monitor': {
+    'msmonitor': {
         'packages': ['WMCore.MicroService.MSMonitor+'],
-        'systems': ['reqmgr2ms-core'],
+        'systems': ['mscore'],
     },
-    'reqmgr2ms-rulecleaner': {
+    'msrulecleaner': {
         'packages': ['WMCore.MicroService.MSRuleCleaner+'],
-        'systems': ['reqmgr2ms-core'],
+        'systems': ['mscore'],
     },
-    'global-workqueue': {
-        'packages': ['WMCore.GlobalWorkQueue+', 'WMCore.WorkQueue+',
+    'wmglobalqueue': {
+        'packages': ['WMCore.wmglobalqueue+', 'WMCore.WorkQueue+',
```

Comment:
This change probably mistakenly feel in your sed/update. Please revert.

Reply:
done, indeed it was auto change applied globally.